checkov/cloudformation/context_parser.py

```diff
@@ -130,6 +131,28 @@ def collect_skip_comments(entity_code_lines: List[Tuple[int, str]]) -> List[_Ski
                     skipped_check["bc_id"] = ckv_to_bc_id_mapping.get(skipped_check["id"])
 
                 skipped_checks.append(skipped_check)
+        if resource_config:
+            metadata = resource_config.get("Metadata")
+            if metadata:
+                ckv_skip = metadata.get("checkov", {}).get("skip", [])
+                bc_skip = metadata.get("bridgecrew", {}).get("skip", [])
+                if ckv_skip or bc_skip:
+                    for skip in itertools.chain(ckv_skip, bc_skip):
+                        skip_id = skip.get("id")
+                        skip_comment = skip.get("comment", "No comment provided")
+                        if skip_id is None:
+                            logging.debug("Check suppression is missing key 'id'")
```

Comment:
@gruebel Let's elevate to `info` / `warning` please, this is highly unexpected! Also add more identifiable data to the log to help the suppressor identify the problematic suppression

Reply:
makes sense, let's go with `warning` and if users complain being to noisy, then we can reduce it later 😄